Lib/difflib.py

```diff
@@ -1038,11 +1038,9 @@ def _qformat(self, aline, bline, atags, btags):
 # remaining is that perhaps it was really the case that " volatile"
 # was inserted after "private".  I can live with that <wink>.
 
-import re
-
-def IS_LINE_JUNK(line, pat=re.compile(r"\s*(?:#\s*)?$").match):
+def IS_LINE_JUNK(line, pat=None):
     r"""
-    Return True for ignorable line: iff `line` is blank or contains a single '#'.
+    Return True for ignorable line: if `line` is blank or contains a single '#'.
```

Comment:
This is not a typo. "iff" means "if and only if".

Reply:
The function will also return `True` if `pat(line) is not None` so I think `iff` is inaccurate here?

---

Otherwise, this misunderstanding has come up before in another file: https://github.com/python/cpython/pull/106776.

And in that case let's use full English words instead: "if and only if".

Reply:
What pat? It is not documented, this is an implementation detail.

Reply:
So I think it's worth reverting

Reply:
Then let's use clear words:
```suggestion
    Return True for ignorable line: if and only if `line` is blank or contains a single '#'.
```

Reply:
TIL about `iff` :)

Reply:
This was unnecessary.

Reply:
@serhiy-storchaka sorry for missing this, it was marked 'resolved', and I didn't see it. I agree changing from 'iff' to 'if' wasn't ideal here, sorry.